### PR TITLE
Take a stab at partial station wording

### PR DESCRIPTION
**Pull request by nonrenewable**
I'm not sure i'm doing this right so will wait to hear from @ironwallaby about thoughts on that but the test sentence should make sense hopefully!

### CODE REVIEW

File: lib/lang/en.js [hunk 1]
```diff
@@ -163,6 +163,9 @@ module.exports = {
 
     return str;
   },
-  "unavailable-radar": "Next hour forecasts are temporarily unavailable due to $1.",
+  "unavailable-radar": "Next hour forecasts are $1 unavailable due to $2.",
+  "temporarily": "temporarily",
+  "partially": "partially",
   "station-offline": "all nearby radar stations being offline",
+  "station-spotty": "nearby radar stations reporting incomplete data sets",
```
Comment (ironwallaby) [inline lib/lang/en.js]: I would prefer `"station-incomplete": "gaps in coverage from nearby radar stations"`

Comment (ironwallaby) [reply lib/lang/en.js]: (Please sanity check with @w1bw @novall @cailees )

Comment (novall) [reply lib/lang/en.js]: I also prefer "gaps in coverage from nearby radar stations".

**Comment by nonrenewable** [comment]
you're a way better person than me 😅  looks good to me! i'll merge it in later today, thank you <3

**Comment by nonrenewable** [comment]
I merge-squashed this all in, so closing.

**Pull request closed without merging**